wormhole-connect/src/store/transferInput.ts

```diff
@@ -273,6 +273,7 @@ const establishRoute = (state: TransferInputState) => {
     Route.NttManual,
     Route.Relay,
     Route.Bridge,
+    Route.Mayan,
```

Comment:
This is a temporary change to unblock the UI testing as we eventually get rid of `routeOrderOfPreference` and rank order the routes according to ETA.